automated install/basic-install.sh

```diff
@@ -1385,6 +1395,11 @@ installConfigs() {
     if [[ -d '/run/systemd/system' ]]; then
         install -T -m 0644 "${PI_HOLE_LOCAL_REPO}/advanced/Templates/pihole-FTL.systemd" '/etc/systemd/system/pihole-FTL.service'
 
+        # Set net admin permissions so that FTL can serve DNS, DHCP and IMAP (for DHCPv6). If this does not work, run FTL as root user.
```

Comment:
Did you re-add this intentionally? It was removed by https://github.com/pi-hole/pi-hole/pull/5043